coordinator/tributary/src/blockchain.rs

```diff
@@ -65,17 +69,33 @@ impl<D: Db, T: TransactionTrait> Blockchain<D, T> {
       block_number: 0,
       tip: genesis,
       next_nonces,
+      unsigned_included: vec![],
```

Comment:
Why is this here?

1) The mempool checks it for a TX's existence. That means it should be a HashSet, not a vec, if kept.
2) We shouldn't store in RAM a full list of every unsigned TX of all time. This should remain at the DB level. Pass the Mempool a lambda, or have it wrap its DB handle with BlockchainDb to get access to the function in BcDb.